src/main/kotlin/no/nav/lydia/ia/sak/domene/IASak.kt

```diff
@@ -49,7 +44,10 @@ class IASak(
                 tilstand.vurderes()
             }
             VIRKSOMHET_ER_IKKE_AKTUELL -> {
-                tilstand.ikkeAktuell()
+                when (hendelse) {
+                    is VirksomhetIkkeAktuellHendelse -> tilstand.behandleHendelse(hendelse = hendelse)
+                    else -> tilstand.ikkeAktuell() // TODO...
```

Comment:
Synes det er mer semantisk å kalle `tilstand.ikkeAktuell` med selve hendelsen i stedet for å ha en `behandleHendelse` funksjon på tilstanden